### PR TITLE
Restore node 14.x support

### PR DESCRIPTION
**Pull request by jrjohnson**
Now that embroider is better up to date across our dependencies we no
longer need the override option in package.json and can relax our
required node version.

### CODE REVIEW

**Comment by jrjohnson** [comment]
This can be merged with the missing test, once it is merged we'll change the requirements for what tests are required.

**Review by stopfstedt — APPROVED** [review]

**Pull request merged**
